ramls/check-in-by-barcode-response.json

```diff
@@ -99,6 +99,10 @@
         }
       },
       "additionalProperties": false
+    },
+    "staffSlipContext": {
```

Comment:
please update raml and descriptor minor versions

Reply:
Done